### DISCUSSION TITLE
Will LuatOS' Air001 be supported？

### DISCUSSION THREAD

**Discussion by 411956158** [moononournation/Arduino_GFX#311] [General]
Shanghai Hezhou recently launched a new 32-bit MCU and officially supported the arduino platform. Here is some introduction.

Air001 is a TSSOP20 packaged MCU that uses a high performance 32-bit ARM®Cortex®-M0+core with 32Kbytes of Flash and 4Kbytes of RAM. The chip integrates multiple communication peripherals such as USART, IIC and SPI, five 16-bit timers, one 12-bit ADC and two comparators. See air001.cn for details.

I want to know whether it will be supported in the future, although its main frequency is lower, but the low price undoubtedly greatly facilitates our creation, thank you.
By the way, i' d like to send you a development board for the air001 if you need it, it doesn' t even need an external crystal (HSE) to run (just using HSI/LSI ), which is very convenient in a small design.![509b7c36-8ca5-47fa-adc5-114375e829fa.png](https://github.com/moononournation/Arduino_GFX/assets/67280906/eecd6e5c-3e35-4be0-9341-c351b6bd378b)

**Comment by moononournation** [comment]
Is it support Arduino IDE?

**Reply by 411956158** [reply]
Yes, the official has launched the package.

**Reply by 411956158** [reply]
Please see the website(air001.cn) for details.

**Comment by moononournation** [comment]
Yes, it can run with SWSPI and HWSPI. Just a little bit unstable:
https://youtu.be/CxfO9YNi04k

**Reply by HalfSweet** [reply]
That's awesome, I'm an Arduino library contributor for Air001, so if you run into any problems, you can give me feedback.
Also, would you be interested in trying the [Air32](https://air32.cn) of the same brand, which has up to 256M mains frequency and theoretically up to 100M SPI output. If you want, please feel free to contact me to give you some samples.
